include/micm/process/cuda_process_set.hpp

```diff
@@ -26,10 +25,17 @@ namespace micm
 #ifdef USE_CUDA
```

Comment:
why not wrap the entire class in the `USE_CUDA` block? I'm just wondering about the case where someone doesn't compile in CUDA support but then is able to create and use a `CudaProcessSet`.

Reply:
done!